packages/tasks/src/model-libraries.ts

```diff
@@ -1040,6 +1040,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"model_vae_fp16.pt"`,
 		snippets: snippets.threedtopia_xl,
 	},
+	acestep: {
+		prettyLabel: "ACE-Step",
+		repoName: "ACE-Step",
+		repoUrl: "https://github.com/ace-step/ACE-Step",
+		filter: false,
+		countDownloads: `path:"ace_step_transformer/diffusion_pytorch_model.safetensors"`,
```

Comment:
perhaps easier would be that we track it via the `config.json`: https://huggingface.co/ACE-Step/ACE-Step-v1-3.5B/blob/main/config.json